src/screens/Compose/components/useAttachments.tsx

```diff
@@ -71,7 +71,7 @@ export default (initialAttachments?: Attachment[]): AttachmentOptions => {
 
   const attachmentsTags = attachments.length > 0 && (
     <View style={styles.attachmentContainer}>
-      {attachments.map((attachment, index) => (
+      {attachments?.map((attachment, index) => (
```

Comment:
In case of undefined attachments we will have an error on first condition, I suggest to add validation before first condition